usbsdmux/usb2642.py

```diff
@@ -450,3 +454,27 @@ def write_to(self, i2cAddr, data):
             raise I2cTransactionFailed(
                 "SCSI-Transaction ended with status {}. I2C-Transaction has probably failed.".format(sgio.status)
             )
+
+    def _read_register(self, reg, size):
```

Comment:
Even if it is an internal method: Please add a short comment what registers it reads from where. Makes maintenance in a few years a lot easier.